src/test/java/com/powsybl/network/conversion/server/NetworkConversionTest.java

```diff
@@ -117,25 +131,26 @@ public void test() throws Exception {
             given(reportServerRest.exchange(eq("/v1/reports/" + reportUuid), eq(HttpMethod.PUT), any(HttpEntity.class), eq(ReporterModel.class)))
                 .willReturn(new ResponseEntity<>(HttpStatus.OK));
 
-            MvcResult mvcResult = mvc.perform(post("/v1/networks")
+            mvc.perform(post("/v1/networks")
                 .param("caseUuid", UUID.randomUUID().toString())
-                .param("reportUuid", UUID.randomUUID().toString()))
+                .param("reportUuid", UUID.randomUUID().toString())
+                .param("isAsyncRun", "false"))
                 .andExpect(status().isOk())
                 .andReturn();
 
-            assertEquals("{\"networkUuid\":\"" + randomUuid + "\",\"networkId\":\"20140116_0830_2D4_UX1_pst\"}",
```

Comment:
why removing the assertion ? import is done synchronously here, like before ...

Reply:
mixed things up while I was developping
fixing